packages/blocks/src/list-block/list-block.ts

```diff
@@ -91,7 +91,20 @@ export class ListBlockComponent extends NonShadowLitElement {
 
   firstUpdated() {
     this.model.propsUpdated.on(() => this.requestUpdate());
-    this.model.childrenUpdated.on(() => this.requestUpdate());
+    this.model.childrenUpdated.on(() => {
+      if (!this.model.children.length) {
+        this.host.page.updateBlock(this.model, {
```

Comment:
Calling `updateBlock` here will lead to redundant model mutation, which will be recorded into history. During collaboration, the same state may by updated for multiple time. Generally we should only listen to updates in `firstUpdated` 🤔.

Reply:
I think this will be resolved by not sitting the toggle state in the crdt, and using awareness store instead, right?

Reply:
@joebeijing do you think the toggle state should be reset after page refresh?

But no matter which approach we choose, we shouldn't mutate model in component lifecycle hook here.

Reply:
i would prefer that there was a local memory of the toggle states, but for now i only managed to get it to refresh after page refresh